src/unit_tests/os_crypto/md5/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ cmake_minimum_required(VERSION 3.10)
 list(APPEND md5_tests_names "test_md5_op")
 list(APPEND md5_tests_flags "-Wl,--wrap,fopen -Wl,--wrap,fclose -Wl,--wrap,fflush -Wl,--wrap,fread \
                              -Wl,--wrap,fseek -Wl,--wrap,fwrite -Wl,--wrap,remove -Wl,--wrap,fprintf \
-                             -Wl,--wrap,fgets -Wl,--wrap,fgetpos -Wl,--wrap,fgetc")
+                             -Wl,--wrap,fgets -Wl,--wrap,fgetpos -Wl,--wrap,fgetc -Wl,--wrap,popen")
```

Comment:
The wrap for function "popen" is not needed in this test, since it is not used by the called functions in it.

Reply:
Yes, it is necessary because the `__wrap_popen` mock was modified to use test_mode, and it must be added so that it finds `__real_popen`, although it is not used directly, if there are `popen` calls in the test

Reply:
Ditto

Reply:
As discussed in the huddle, it was determined that they were necessary.